cabby/model/text/dual_encoder/dataset_rvs.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple, Sequence, Optional, Dict, Text, Any
```

Comment:
Two things:

(1) Please alphabetize the imports.
(2) Make sure you are using all of these in this file. If not, don't include them.

It's trivial, but it helps us reason more quickly about these files in the future. Thanks!